### PR TITLE
LMB-391: Change rangorde up and down of mandataris in orgaan

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We want to change the rangorde of a mandataris in orgaan to be moved up or down in the table. The input field is a free text input but we are setting some limitations on it warning users of potential issues.

* We support changing the rangorde using buttons from 1 to 20 inclusive
* Gibberish text is allowed but shows a warning
* A rangorde is automatically generated based on the mandate of the mandataris when clicking plus or mandataris
* Sorting works based on the numeric value of the rangorde and is unchanged for other columns
* when changing the rangorde using the buttons, the mandataris switches rangorde with the mandataris that previously had that rangorde
* changing rangorde also takes into account the sorting

## How to test

Check the rangorde of the mandaten

## Attachments

image

### CODE REVIEW

**Review by Rahien — CHANGES_REQUESTED** [review]
need to be able to enter other things than numbers.

**Comment by ruben-bottu** [comment]
Functionality wise looks good

File: app/components/verkiezingen/rangorde-input.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  }
+
+  findOrderInString(possibleString) {
+    if (!possibleString) {
+      return null;
+    }
+    let foundNumber = null;
+    Object.keys(rangordeStringMapping).forEach((key) => {
+      if (possibleString.startsWith(key)) {
+        foundNumber = rangordeStringMapping[key];
```
Comment (ruben-bottu) [inline app/components/verkiezingen/rangorde-input.js:65]: Maybe the loop should be ended early if a match is found?

**Review by ruben-bottu — APPROVED** [review]
Looks good

**Review by Rahien — APPROVED** [review]

**Pull request merged**